content/en/blog/_posts/2021-XX-XX-writing-a-controller-for-pod-labels.md

```diff
@@ -0,0 +1,473 @@
+---
+layout: blog
+title: "Writing a Controller for Pod Labels"
+date: 2021-03-XX
```

Comment:
Set the date.

Reply:
What date should I use? 🤔

Reply:
@sftim Can you answer this question?

Reply:
@busser - Please update this with the current date. For example, `date: 2021-04-29`. (Okay @sftim?)

Reply:
I've set the date to `2021-05-04` 👍